<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrustView</class>
 <widget class="QWidget" name="TrustView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>394</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="trustTableTW">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Zoom :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="zoomHS">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updatePB">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="showingLabel">
       <property name="text">
        <string>Showing: whole network</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="3"/>
 <resources/>
 <connections/>
</ui>
